ZenPacks/zenoss/ZenPackLib/lib/spec/RRDThresholdSpec.py

```diff
@@ -62,6 +65,7 @@ def __init__(
             self.extra_params = {}
         else:
             self.extra_params = extra_params
+        self.optional = optional
```

Comment:
Should probably move it before extra_params here too. Just for consistency. Obviously it doesn't matter for functionality.